README.md

```diff
@@ -1,23 +1,24 @@
-# Accurate Instagram Filters Reconstruction
+# Accurate Image Filters Reconstruction
 
 There's a bunch of apps out there (e.g., Instagram) allowing you to apply color filters to images.
 You might be interested in cloning their behavior: reconstruct them. This
 repository holds tools for semi-automatically and very accurately reconstructing
 color filters.
 
-The truth is folks like Instagram filters. They are trying to
+The truth is folks like filters. They are trying to
 [reproduce](https://github.com/girliemac/Filterous)
 them [again](https://github.com/girliemac/filterous-2)
 and [again](https://github.com/acoomans/instagram-filters).
 And [again](https://github.com/lukexyz/CV-Instagram-Filters)
 and [again](https://www.practicepython.org/blog/2016/12/20/instagram-filters-python.html).
 And [again](https://code.tutsplus.com/tutorials/create-instagram-filters-with-php--net-24504)
 and [again](https://picturepan2.github.io/instagram.css/).
+
 The problem with the attempts is they mostly deal with manually correcting
 colors.
+While, there's just a single robust method of color reconstruction.
 
-This is the only accurate reconstruction of color filters using a robust method.
-For instance, one of the following images was obtained using Instagram filter,
+For instance, one of the following images was obtained using a commercial filter,
```

Comment:
There were two aims: show that the method works, show that the method works for Instagram.

Reply:
Well, "Instagram-like", "Instagram filter, as is would be the most popular one"